strawberryfields/io.py

```diff
@@ -40,6 +41,7 @@ def to_blackbird(prog, version="1.0"):
         blackbird.BlackbirdProgram:
     """
     bb = blackbird.BlackbirdProgram(name=prog.name, version=version)
+    bb._modes = list(prog.reg_refs.keys())
```

Comment:
Was this an oversight that this wasn't being set before?

Reply:
Yes, it wasn't there before. :grimacing: